tests/components/create_team/create_team.test.jsx

```diff
@@ -34,14 +34,6 @@ describe('/components/create_team', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
-    test('should display custom branded description', () => {
-        const wrapper = shallowWithIntl(<CreateTeam {...defaultProps}/>);
-
-        const customText = wrapper.find('h4').text();
-
-        expect(customText).toBe(defaultProps.customDescriptionText);
-    });
```

Comment:
Removed this test since there is covered by `tests/components/common/site_name_and_description.test.jsx`